src/licensedcode/data/rules/jj2000_1.RULE

```diff
@@ -0,0 +1,7 @@
+---
+license_expression: jj2000
+is_license_tag: yes
+---
+
+name: {{JJ2000}}
+url: LICENSE-JJ2000.txt
```

Comment:
You could add a referenced filenames for this

Reply:
Added this thanks!